src/components/__tests__/FooterBar.cy.js

```diff
@@ -5,6 +5,14 @@ import { i18n } from '../../boot/i18n';
 const { getPaletteColor } = colors;
 const white = getPaletteColor('white');
 
+const rideToWorkByBikeConfig = JSON.parse(
+  process.env.RIDE_TO_WORK_BY_BIKE_CONFIG,
+);
+const urlYoutube = rideToWorkByBikeConfig.urlYoutube;
```

Comment:
You can move this `const` declaration into ./src/boot/global_vars.js file.

Reply:
Good point. I used the global `rideToWorkByBikeConfig` export from boot file and imported to test and component. 
I also changed boot file to `.ts` file which will provide types for the config object. 
Because of that, it seems unnecessary to provide exports for each individual config property - the `const` declaration you mentioned in test file is replaced by using the property of the config object.